dist/profile/manifests/jiraveg.pp

```diff
@@ -0,0 +1,32 @@
+# profile for jira
+class profile::jiraveg (
```

Comment:
You can name like you want, like a man who has many servers vegetable code is not remembered for me, so i mixed jira+vegetable = jiraveg.